uaclient/api/api.py

```diff
@@ -21,6 +21,7 @@
     "u.pro.attach.magic.wait.v1",
     "u.pro.attach.magic.revoke.v1",
     "u.pro.attach.auto.should_auto_attach.v1",
+    "u.pro.security_status.packages.v1",
```

Comment:
api endpoint name bikeshedding thread:

While related to ESM (which is why we're doing this at all), it isn't returning information directly related to pro features. So I don't think it should be in the "pro" namespace.
It should probably be in some namespace, maybe a generic "security".
The method name should be specific for this format, something like "get_package_manifest"

So I suggest: `u.security.get_package_manifest`

Reply:
makes sense, even making more sense why the integration tests should just be run on containers

Reply:
I would even remove the `get`, leaving it `u.security.package_manifest.v1` but it's just an opinion.